README.md

```diff
@@ -6,6 +6,34 @@ Fissile converts existing BOSH dev releases into docker images.
 It does this using just the releases, without a BOSH deployment, CPIs, or a BOSH
 agent.
 
+## Build and Install
+Building fissile needs Go 1.4 or higher. You can download it from [website](https://golang.org/doc/install)
```

Comment:
Final nitpick from me, `website` is too generic. How about 'the Golang website'?

PS. Thanks for adding docs, we appreciate it!